src/main/java/frc/robot/Constants.java

```diff
@@ -17,12 +17,27 @@
  */
 public final class Constants {
     public final static class Ports {
-        public static final int FRONT_LEFT_MOTOR_PORT = 0;
+        public static final int FRONT_LEFT_MOTOR_PORT = 4;
         public static final int FRONT_RIGHT_MOTOR_PORT = 1;
-        public static final int REAR_LEFT_MOTOR_PORT = 2;
-        public static final int REAR_RIGHT_MOTOR_PORT = 3;
+        public static final int REAR_LEFT_MOTOR_PORT = 3;
+        public static final int REAR_RIGHT_MOTOR_PORT = 2;
+
+        // public static final int FRONT_LEFT_MOTOR_PORT = 3;
+        // public static final int FRONT_RIGHT_MOTOR_PORT = 5;
+        // public static final int REAR_LEFT_MOTOR_PORT = 2;
+        // public static final int REAR_RIGHT_MOTOR_PORT = 4;
         
         public static final int XBOX_CONTROLLER_PORT = 0;
-    }    
+
+        public static final int LEFT_ENCODER_MOTOR = 0;
+        public static final int RIGHT_ENCODER_MOTOR = 1;
+        public static final int RIGHT_ENCODER_A = 2;
+        public static final int RIGHT_ENCODER_B = 3;
+    }
+
+    // in inches
+    public static final double WHEEL_CIRCUMFERENCE = 6d * Math.PI;
+    public static final double ENCODER_COUNTS_PER_REVOLUTION = 4096;
+    public static final double kEncoderDistancePerPulse = WHEEL_CIRCUMFERENCE / ENCODER_COUNTS_PER_REVOLUTION;
```

Comment:
These three constants can go in `Drivetrain`, not here, unless they have to be used all over or changed frequently. Also, match formatting (i.e. ENCODER_DISTANCE_PER_PULSE)